addon/plugins/rdfa-date-plugin/nodes/date.ts

```diff
@@ -16,11 +16,9 @@ const date: (options: DateOptions) => NodeSpec = (options) => {
       mappingResource: {
         default: null,
       },
-      value: {
-        default: new Date().toISOString(),
-      },
+      value: {},
       format: {
-        default: 'dd/MM/yyyy',
+        default: options.formats[0].dateFormat,
```

Comment:
Inserting a date in the dummy app is broken, I think it's cause of this line

Reply:
Should be fixed now.